torch/_inductor/lowering.py

```diff
@@ -6219,10 +6220,199 @@ def while_loop(cond_fn, body_fn, carried_inputs, additional_inputs):
             msg = f"{msg} Found from : \n {stack_trace}"
         V.graph.disable_cudagraphs_reason = msg
 
+    fx_all_inputs = V.graph.current_node.args[-2] + V.graph.current_node.args[-1]  # type: ignore[operator]
```

Comment:
Moved the lowering of subgraph sfrom WhileLoop.create here. The purpose is to incorporate scan's special lowering logic.